joomla/joomla-cms
# [4.2] Load the actionlog params from model instead of the helper

### Summary of Changes
Move the logic to fetch the log content type params of the action log component to a model. Models should do the database calls as they have access to it. Like that it is possible to inject the database object and it is not necessary to do a deprecated `Factory::getDbo()` call. This helps to write proper unit tests which improves the sustainability. Having the code tested with unit tests ensures that errors in further changes will be detected and potential BC breaks can be eliminated. This is only a small step but we need to head into that direction when we want to have a system where we can write easy system tests without a bunch of preparation setup.

Additionally it makes a small modification to the extension namespace mapper so it can be loaded during the bootstrap process of the unit test suite, similar to the libraries.

### Testing Instructions
- Make sure the Joomla action log plugin is loaded
- Save an article

### Actual result BEFORE applying this Pull Request
An entry is written to the user action log that an article is saved.

### Expected result AFTER applying this Pull Request
An entry is written to the user action log that an article is saved.
